src/main/resources/default/templates/biz/jobs/scheduler/entry.html.pasta

```diff
@@ -68,50 +72,53 @@
                 </i:for>
-            </w:singleSelect>
+            </t:singleSelect>
         </div>
 
-        <w:subHeading labelKey="SchedulerController.planning"/>
+        <t:heading labelKey="SchedulerController.planning"/>
         <div class="row">
-            <w:textfield name="schedulerData_year"
+            <t:textfield class="col-md"
+                         name="schedulerData_year"
                          labelKey="SchedulerData.year"
                          helpKey="SchedulerData.year.help"
                          value="@entry.getSchedulerData().getYear()"/>
-            <w:textfield name="schedulerData_month"
+            <t:textfield  class="col-md"
+                          name="schedulerData_month"
                          labelKey="SchedulerData.month"
                          helpKey="SchedulerData.month.help"
                          value="@entry.getSchedulerData().getMonth()"/>
-        </div>
-        <div class="row">
-            <w:textfield name="schedulerData_dayOfMonth"
+            <t:textfield class="col-md"
+                         name="schedulerData_dayOfMonth"
                          labelKey="SchedulerData.dayOfMonth"
                          helpKey="SchedulerData.dayOfMonth.help"
                          value="@entry.getSchedulerData().getDayOfMonth()"/>
-            <w:textfield name="schedulerData_dayOfWeek"
+        </div>
+        <div class="row">
+            <t:textfield class="col-md"
+                         name="schedulerData_dayOfWeek"
                          labelKey="SchedulerData.dayOfWeek"
                          helpKey="SchedulerData.dayOfWeek.help"
                          value="@entry.getSchedulerData().getDayOfWeek()"/>
-        </div>
-        <div class="row">
-            <w:textfield name="schedulerData_hourOfDay"
+            <t:textfield class="col-md"
+                         name="schedulerData_hourOfDay"
                          labelKey="SchedulerData.hourOfDay"
                          helpKey="SchedulerData.hourOfDay.help"
                          value="@entry.getSchedulerData().getHourOfDay()"/>
-            <w:textfield name="schedulerData_minute"
+            <t:textfield class="col-md"
+                         name="schedulerData_minute"
                          labelKey="SchedulerData.minute"
                          helpKey="SchedulerData.minute.help"
                          value="@entry.getSchedulerData().getMinute()"/>
         </div>
-        <w:subHeading labelKey="SchedulerController.parameters"/>
+        <t:heading labelKey="SchedulerController.parameters"/>
         <div class="row">
             <i:for type="sirius.biz.jobs.params.Parameter" var="param"
                    items="entry.getJobConfigData().getJobFactory().getParameters()">
                 <i:dynamicInvoke template="@param.getTemplateName()" param="@param"
                                  context="@entry.getJobConfigData().getConfigMap()"/>
```

Comment:
CodeStyle: if not all parameters fit into one line, use one line for each parameter for better readability.
```suggestion
            <i:for type="sirius.biz.jobs.params.Parameter" 
                   var="param"
                   items="entry.getJobConfigData().getJobFactory().getParameters()">
                <i:dynamicInvoke template="@param.getTemplateName()"
                                 param="@param"
                                 context="@entry.getJobConfigData().getConfigMap()"/>
```